packages/decimal/decimal.0.3.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Arbitrary-precision floating-point decimal library"
+description: """
+Arbitrary-precision floating-point decimal library ported from
+the Python decimal module."""
+maintainer: ["Yawar Amin <yawar.amin@gmail.com>"]
+authors: ["Yawar Amin <yawar.amin@gmail.com>"]
+license: "PSF-2.0"
+homepage: "https://github.com/yawaramin/ocaml-decimal"
+doc: "https://yawaramin.github.io/ocaml-decimal/api"
+bug-reports: "https://github.com/yawaramin/ocaml-decimal/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "alcotest" {>= "1.5.0" & < "2.0.0" & with-test}
+  "angstrom" {>= "0.15.0" & < "1.0.0" & with-test}
+  "ocaml" {>= "4.08.0"}
+  "zarith" {>= "1.10" & < "2.0.0"}
```

Comment:
Are the upper bounds needed or we can drop them for the time being?

Reply:
I realize it's not perfect, but I'm attempting to prevent breaking builds in case one of the library dependencies increments a major version and makes a breaking change. I would prefer to keep the upper bounds.

Reply:
I can keep them, but we have been adding them consistently each time a package broke something in the revdeps (before the merge!)

Reply:
Also more often than not breakages came also before a major release

Reply:
Understood, let's keep it for now just as a precaution even though we realize it's not perfect.